test/subscribe.test.js

```diff
@@ -0,0 +1,89 @@
+describe('Subscribe', function() {
+  describe('PersistedModel', function() {
+    describe('configured to source changes locally', function() {
+      before(function() {
+        var test = this;
+        var app = loopback({localRegistry: true});
+        var ds = app.dataSource('ds', {connector: 'memory'});
+        this.Score = app.model('Score', {
+          dataSource: 'ds',
+          changeDataSource: false // use only local observers
```

Comment:
Can we use a more specific name? Consider the existing change tracking & synchronization algorithm. I can imagine that we may want to attach model's Change object to a different datasource than the model (e.g. Car is in Oracle, Car-Changes is in MongoDB). In which case I would expect that the setting `changeDataSource` refers to the datasource used by the Change model.

Perhaps we can omit this placeholder altogether if there is no real implementation yet?